packages/stark-build/config/webpack.common.js

```diff
@@ -166,14 +166,16 @@ module.exports = options => {
 				// see https://github.com/angular/angular-cli/issues/8594
 				{
 					test: /(?:\.ngfactory\.js|\.ngstyle\.js|\.ts)$/,
-					use: METADATA.AOT ? [buildOptimizerLoader, "@ngtools/webpack"] : ["@ngtools/webpack"]
+					use: METADATA.AOT ? [buildOptimizerLoader, "@ngtools/webpack"] : ["@ngtools/webpack"],
+					exclude: METADATA.AOT ? [helpers.root("node_modules/prettier")] : []
 				},
 
 				...(METADATA.AOT
 					? [
 							{
 								test: /\.js$/,
-								use: [buildOptimizerLoader]
+								use: [buildOptimizerLoader],
+								exclude: [helpers.root("node_modules/prettier")]
```

Comment:
Actually, this exclude is not needed to make the build working. Could you please remove it ? 😊

Reply:
It's gone

Reply:
@christophercr ask me to add them again, because without these excludes we have "circular dependency" warnings in the build.

Reply:
Alright !